modules/tensor_mechanics/test/tests/beam/dynamic/tests

```diff
@@ -1,173 +1,287 @@
 [Tests]
+  design = "C0TimoshenkoBeam.md"
+  issues = "#10313"
   [./dyn_euler]
     type = 'Exodiff'
     input = 'dyn_euler_small.i'
     exodiff = 'dyn_euler_small_out.e'
+
+    requirement = "The dynamic system shall correctly predict the natural frequencies"
+                  " of an Euler-Bernoulli beam with consistent mass/inertia."
   [../]
   [./dyn_timoshenko]
     type = 'Exodiff'
     input = 'dyn_timoshenko_small.i'
     exodiff = 'dyn_timoshenko_small_out.e'
+
+    requirement = "The dynamic system shall correctly predict the natural frequencies"
+                  " of a Timoshenko beam with consistent mass/inertia."
```

Comment:
The mechanics system shall correctly predict the natural frequencies of a Timoshenko beam modeled using beam elements with consistent mass/inertia.